source/usage-examples/findOne.txt

```diff
@@ -14,35 +14,50 @@ Example
 
 .. include:: /includes/usage-examples/run-example-tip.rst
 
-The following example matches documents in the ``movies`` collection
-in which the ``title`` is "The Room", returning the first document
+This example uses the following ``Restaurant`` struct as a model for documents
+in the ``restaurants`` collection:
+
+.. literalinclude:: /includes/usage-examples/code-snippets/findOne.go
+   :start-after: start-restaurant-struct
+   :end-before: end-restaurant-struct
+   :language: go 
+   :copyable: 
+   :dedent:
+
+The following example matches documents in the ``restaurants`` collection
+in which the ``name`` is "Bagels N Buns", returning the first document
 matched:
 
 .. literalinclude:: /includes/usage-examples/code-snippets/findOne.go
    :start-after: begin findOne
    :end-before: end findOne
    :language: go
    :dedent:
-   :emphasize-lines: 4
+   :emphasize-lines: 5
 
 View a `fully runnable example <{+example+}/findOne.go>`__
 
 Expected Result
 ---------------
 
-After you run the full example, it returns a ``SingleResult``
-object that contains the following document:
+Running the full example prints the following document, which is stored in the 
+``result`` variable as a ``Restaurant`` struct:
 
 .. code-block:: json
```

Comment:
I'm not sure if the code should print the document as a struct or in json, as it currently does. It looks pretty messy as a struct because of the address and grades fields. What are your thoughts?

Reply:
I think we keeping it in JSON will be best for users — struct is a bit hard on the eyes, especially as we deal w/ field values that have their own spaces and whatnot. Will adjust the language to specify that it's being printed as a json!